resources/public/api/conf/1.0/checkpoints_list.yaml

```diff
@@ -0,0 +1,49 @@
+get:
+  $ref: "./checkpoints_list.yaml"
+  summary: List checkpoints
+  description: |
+    This endpoint allows a developer to get all the checkpoints created for the vendor ID. 
+    A National Insurance number can be provided to filter the results.
+  security:
+    - User-Restricted:
+        - write:self-assessment
+  parameters:
+    - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+    - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
+    - $ref: './common/queryParameters.yaml#/components/parameters/ninoCreateCheckpoint'
+
+  responses:
+    200:
+      description: Success
+      headers:
+        X-CorrelationId:
+          $ref: './common/headers.yaml#/components/parameters/correlationId'
+      content:
+        application/json:
+          schema:
+            $ref: './schemas/checkpoints/list_response.json'
+          examples:
+            example-response:
+              description: Example Response
+              value:
+                $ref: './examples/checkpoints/list_response.json'
+
```

Comment:
The 400 and 403 errors are not displaying atm: 
<img width="520" alt="image" src="https://github.com/hmrc/mtd-sa-test-support-api/assets/106095575/5464bda9-a3dd-4b06-a76b-c7e682c8f5ab">